src/resources/views/crud/columns/time.blade.php

```diff
@@ -0,0 +1,4 @@
+@php
+    $column['format'] = $column['format'] ?? 'HH:mm';
```

Comment:
## time, week, month and date doesn't use the same format as the field
![image](https://user-images.githubusercontent.com/7188159/198280682-14c4026e-2f65-4ec9-a4a1-c1836e51c00a.png)
![image](https://user-images.githubusercontent.com/7188159/198280536-d4af2a07-f1d9-4913-9714-78cc48ea794d.png)